src/site/content/en/blog/css-props-and-vals/index.md

```diff
@@ -0,0 +1,206 @@
+CSS. While incredibly useful today, they have shortcomings that can make them
+hard to work with: they can take any value so they may be accidentally
+overridden with something unexpected, they always inherit their values from
+their parent, and you can't transition them. With Houdini's [CSS Properties and
+Values API Level 1](https://drafts.css-houdini.org/css-properties-values-api/),
+now available in Chrome 78, ßthese shortcomings are transcended, making CSS
+custom properties incredibly powerful!
+
+## What Is Houdini?
+
+Before talking about the new API, let's talk about Houdini quickly. The CSS-TAG
+Houdini Task Force, better known as CSS Houdini or simply Houdini, exists to
+"develop features that explain the 'magic' of styling and layout on the web".
+The collection of [Houdini specifications](https://drafts.css-houdini.org/) are
+designed to open up the power of the browser's rendering engine, allowing both
+deeper insight into our styles and the ability to extend our rendering engine.
+With this, typed CSS values in JavaScript and polyfilling or inventing new CSS
+without a performance hit are finally possible. Houdini has the potential to
+superpower creativity on the web.
+
+## CSS Properties and Values API Level 1
+
+The CSS Properties and Values API Level 1 (Houdini Props and Vals) allows us to
+give structure to our custom properties. This is the current situation when
+using custom properties:
+
+```css
+.thing {
+  --my-color: green;
+}
+```
+
+Because custom properties don't have types, they can be overridden in unexpected
+ways. For example, consider what happens if you define `--my-color` with a URL.
+
+```css
+.thing {
+  --my-color: url(‘not-a-color’);
+  color: var(--my-color);
+}
+```
+
+Here, because `--my-color` isn't typed, it doesn't know that a URL isn't a valid
+color value! When we use it, it falls back to default values (black for `color`,
+transparent for `background`). With Houdini Props and Vals, custom properties can
+be _registered_ so that the browser knows what it _should_ be!
+
+Now, the custom property `--my-color` is registered as a color! This tells the
+browser what kinds of values are allowed and how it can type and treat that
+property!
+
+### Anatomy of a registered property
+
+`registerProperty()` takes four options:
+
+<dl>
+  <dt><code>name</code></dt>
+  <dd>The name of the custom property. String.</dd>
+  <dt><code>syntax</code></dt>
+  <dd>How to parse the custom property. You can find a complete list of possible values in the <a href="https://drafts.csswg.org/css-values-3/">CSS Values and Units</a> specification. Defaults to <code>*</code>. String.</dd>
+  <dt><code>inherits</code></dt>
+  <dd>Whether it inherits its parent’s value. Defaults to <code>true</code>. Boolean.</dd>
+  <dt><code>initialValue</code></dt>
+  <dd>Initial value of the custom property. String.</dd>
+</dl>
+
+Registering a property looks like this:
```

Comment:
Just curious if you might want to put this before the text that appears at line 79. I noticed that line 79 starts in with `registerProperty()` but that's the first time it's mentioned in the article so it is sort of lacking a bit of context.

Another option would be to add a sentence before line 79 that says something like, "To register a property, use the `registerProperty()` method in your JavaScript. `registerProperty()` takes four options:..."